src/main/java/uk/gov/justice/hmpps/prison/api/model/AssessmentDetail.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Assessment Detail
+ **/
+@SuppressWarnings("unused")
+@ApiModel(description = "AssessmentDetail")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class AssessmentDetail {
```

Comment:
We did consider extending the Assessment API object but the project team identified some properties in the other object, e.g. overrideReason, that are misleading. Therefore we made a new one.

Also, we did consider using "CSRA" in the name, but I think this will be useful for all assessments going forward.